app/models/series.rb

```diff
@@ -39,6 +39,7 @@ def title
     too_long: ts("must be less than %{max} letters long.", max: ArchiveConfig.NOTES_MAX)
 
   after_save :adjust_restricted
+  after_update_commit :update_work_index
```

Comment:
Today I Learned about `after_update_commit` 🤓💥

Reply:
We found that we should be using `after_commit` for ES callbacks (like #3300). Ideally. Eventually.